src/associated_token_account.zig

```diff
@@ -2,7 +2,8 @@ const std = @import("std");
 const bincode = @import("bincode");
 const sol = @import("solana-program-sdk");
 
-pub const program_id = sol.PublicKey.comptimeFromBase58("ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL");
+const AssociatedTokenProgram = @This();
```

Comment:
nit: the program is called the associated token *account* program
```suggestion
const AssociatedTokenAccountProgram = @This();
```

Reply:
done ✅